backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Helpers/PaginationUtils.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Altinn.AccessManagement.UI.Core.Models.ResourceRegistry;
+
+namespace Altinn.AccessManagement.UI.Core.Helpers
+{
+    /// <summary>
+    /// Utils for pagination
+    /// </summary>
+    internal static class PaginationUtils
+    {
+        /// <summary>
+        /// Paginates a list, returning the list elements pertaining to the given page
+        /// </summary>
+        /// <typeparam name="T">The type contained in the list</typeparam>
+        /// <param name="list">The list to be paginated</param>
+        /// <param name="page">The page number of the returned elements </param>
+        /// <param name="numPerPage">Number of elements on each page</param>
+        /// <returns>The elements present on the given page</returns>
+        /// <exception cref="ArgumentOutOfRangeException">This exception is thown when either numPerage or page is not valid, souch as if asking for a page does not exist</exception>
+        public static PaginatedList<T> GetListPage<T>(List<T> list, int page, int numPerPage)
```

Comment:
I would think that it can be safer and perhaps more efficient to just use LINQ for the pagination.
Something like: 
`list.Skip(page*pageSize).Take(pageSize).ToList()`

Have you looked into something like that? Might still need some logic to handle/check argument sizes against the size of the list to avoid out of bounds issues.

Reply:
Not sure what you mean here..
Like instead of GetRange? They basically do the same don't they? 😅

I just did a quick check on StackOverflow and according to the accepted answer there:
"The only difference is that List.GetRange is more efficient than Take(n).ToList() since it already knows the size of the new list whereas the LINQ methods don't know it's size."

Reply:
Interesting. The StackOverflow aswer is getting a bit old though. LINQ has had a lot of optimization lately.

It was mostly a thought of whether there was an actual need for a PaginatedList model / Util and custom logic if it could be solved by LINQ directly. Otherwise I don't see any issue with your implementation.